framework/Optimizers/GradientBasedOptimizer.py

```diff
@@ -438,17 +478,18 @@ def localFinalizeActualSampling(self,jobObject,model,myInput):
     # for some reason, Ensemble Model doesn't preserve this information, so wrap this debug in a try:
     try:
       prefix = jobObject.getMetadata()['prefix']
-      self.raiseADebug('Collected sample "{}"'.format(prefix))
     except TypeError:
       prefix = ''
     self.raiseADebug('Collected sample "{}"'.format(prefix))
 
     # TODO REWORK move this whole piece to Optimizer base class as much as possible
+    # TODO what if there's no solution export?  Our "status" and etc fail badly!
```

Comment:
I was too afraid to test this.  I will copy an existing test case and see what happens.